lib/metamodel.cto

```diff
@@ -208,6 +208,10 @@ concept LongDomainValidator {
   o Long upper optional
 }
 
+concept AliasType{
```

Comment:
```suggestion
concept AliasedType {
```
to align with the property with which this definition will be used.

Reply:
done 👍🏻